### PR TITLE
Core Team Pattern - Improvements

### PR DESCRIPTION
**Pull request by spier**
Implements #346 

- Add Core Team to mindmap
   - Technical Challenges => Project is difficult to contribute to and use => Core Team
   - Organization Challenges  => Project without an owner/maintainer => Core Team
- Adds a visual to the pattern 
- Adding separate links for **Learning Path** and **Pattern**

### CODE REVIEW

**Comment by spier** [comment]
@rrrutledge would you mind reviewing these proposed changes?

**Comment by spier** [comment]
@rrrutledge this PR now contains all the changes discussed in #346 , including the visual.

_Note:_ I could not find a good way to embed an SVG of the visual, because gitbook would always zoom in into it in strange ways. Hence I am using a PNG for now but the SVG is also part of this PR, if we want to use it later.

Thanks for the review :)

File: pattern-categorization/innersource-program-mind-map.html [hunk 1]
```diff
@@ -25,6 +25,6 @@
...
...rce License</a>"}]}]}]},{"t":"heading","d":2,"p":{"lines":[94,95]},"v":"Grow","c":[{"t":"heading","d":3,"p":{"lines":[96,97]},"v":"Discovery Challenges","c":[{"t":"heading","d":4,"p":{"lines":[98,99]},"v":"Can't find matching projects","c":[{"t":"heading","d":5,"p":{"lines":[100,101]},"v":"<a href=\"https://patterns.innersourcecommons.org/p/gig-marketplace\">Gig Marketplace</a>"},{"t":"heading","d":5,"p":{"lines":[102,103]},"v":"<a href=\"https://patterns.innersourcecommons.org/p/innersource-portal\">InnerSource Portal</a>"}]},{"t":"heading","d":4,"p":{"lines":[104,105]},"v":"Difficult to find active projects","c":[{"t":"heading","d":5,"p":{"lines":[106,107]},"v":"<a href=\"https://patterns.innersourcecommons.org/p/repository-activity-score\">Repository Activity Score</a>"}]}]}]},{"t":"heading","d":2,"p":{"lines":[108,109]},"v":"Scale","c":[{"t":"heading","d":3,"p":{"lines":[110,111]},"v":"Self Education/Improvement Challenges","c":[{"t":"heading","d":4,"p":{"lines":[112,113]},"v":"Not aware of InnerSource best practices","c":[{"t":"heading","d":5,"p":{"lines":[114,115]},"v":"<a href=\"https://patterns.innersourcecommons.org/p/maturity-model\">Maturity Model</a>"}]}]}]}]})</script>
```
Comment (rrrutledge) [inline pattern-categorization/innersource-program-mind-map.html:28]: Could be good one day to split into separate lines for easier review.

**Review by rrrutledge — APPROVED** [review]
Thanks for driving these improvements!  At first I was wondering why we needed to push for more work given that it seemed like it was already a good pattern.  Things seemed pretty straightforward already.  But I have to say, these extra changes do make it better, and I'm glad to have them.  Thank you!

File: pattern-categorization/innersource-program-mind-map.html (hunk 1 above)
Comment (spier) [reply pattern-categorization/innersource-program-mind-map.html:28]: The HTML file is autogenerated from `innersource-program-mind-map.md`, so this doesn't have to get reviewed.

**Comment by spier** [comment]
Great to hear that these changes are adding value.
I expect that once other external readers review this pattern, we might even see more feedback. After all, for the authors of a pattern everything tends to be straightforward anyways, as they also know the things that haven't actually written down.

So thanks for contributing to this improvements, and giving things a try, even more so as you weren't convinced initially :)

**Pull request merged**